xdebug_com.c

```diff
@@ -21,169 +21,242 @@
 #include <errno.h>
```

Comment:
I'm using the Windows GitHub desktop app and this change has happend automatically. I will change it back. I actually changed two files back to 755 because after my push the change in file permissions was shown here. But that was two bash files so it was correct.